src/XMakeBuildEngine/Construction/ProjectRootElement.cs

```diff
@@ -1714,8 +1714,18 @@ public void Save(Encoding saveEncoding)
                 {
                     using (ProjectWriter projectWriter = new ProjectWriter(_projectFileLocation.File, saveEncoding))
                     {
-                        projectWriter.Initialize(XmlDocument);
-                        XmlDocument.Save(projectWriter);
+                        var xmlWithNoImplicits = (XmlDocument)XmlDocument.Clone();
```

Comment:
ah, I apparently hadn't committed and pushed that fix when I made it. You're right--the replacement is `CloneNode(true)`.